Index: src/test/java/runner/TestRunner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package runner; // Asegúrate de que coincida con el nombre de tu carpeta\r\n\r\nimport io.cucumber.junit.Cucumber;\r\nimport io.cucumber.junit.CucumberOptions;\r\nimport org.junit.runner.RunWith;\r\n\r\n@RunWith(Cucumber.class)\r\n@CucumberOptions(\r\n        features = \"src/test/resources/features\",\r\n        glue = {\"\"}, // Asegúrate de que el nombre coincida exactamente con el paquete\r\n        plugin = {\"pretty\"}\r\n\r\n        /*Prueba*/\r\n)\r\npublic class TestRunner {\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/runner/TestRunner.java b/src/test/java/runner/TestRunner.java
--- a/src/test/java/runner/TestRunner.java	(revision 7de99e27c8239f7c3da39ee13fbe5df3d596528c)
+++ b/src/test/java/runner/TestRunner.java	(date 1772079306328)
@@ -10,7 +10,6 @@
         glue = {""}, // Asegúrate de que el nombre coincida exactamente con el paquete
         plugin = {"pretty"}
 
-        /*Prueba*/
 )
 public class TestRunner {
 }
\ No newline at end of file
