Index: src/test/java/runner/TestRunner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package runner; // Asegúrate de que coincida con el nombre de tu carpeta\r\n\r\nimport io.cucumber.junit.Cucumber;\r\nimport io.cucumber.junit.CucumberOptions;\r\nimport org.junit.runner.RunWith;\r\n\r\n@RunWith(Cucumber.class)\r\n@CucumberOptions(\r\n        features = \"src/test/resources/features\",\r\n        glue = {\"\"}, // Asegúrate de que el nombre coincida exactamente con el paquete\r\n        plugin = {\"pretty\"}\r\n(*prueba*/)\r\n)\r\npublic class TestRunner {\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/runner/TestRunner.java b/src/test/java/runner/TestRunner.java
--- a/src/test/java/runner/TestRunner.java	(revision 1f32213133c6aee6162a862ed0864680d845cfad)
+++ b/src/test/java/runner/TestRunner.java	(date 1772091539860)
@@ -9,7 +9,7 @@
         features = "src/test/resources/features",
         glue = {""}, // Asegúrate de que el nombre coincida exactamente con el paquete
         plugin = {"pretty"}
-(*prueba*/)
+/*prueba*/)
 )
 public class TestRunner {
 }
\ No newline at end of file
